src/main/java/org/jenkinsci/plugins/pipeline/github/trigger/GitHubParametersAction.java

```diff
@@ -0,0 +1,68 @@
+package org.jenkinsci.plugins.pipeline.github.trigger;
+
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.model.EnvironmentContributor;
+import hudson.model.ParameterValue;
+import hudson.model.ParametersAction;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+
+@Restricted(NoExternalUse.class)
+public class GitHubParametersAction extends ParametersAction {
```

Comment:
Maybe call this GitHubEnvironmentVariablesAction to more closely match what it's doing?